src/Templates/src/templates/maui-mobile/MainPage.xaml

```diff
@@ -4,37 +4,32 @@
              x:Class="MauiApp._1.MainPage">
 
     <ScrollView>
-        <VerticalStackLayout
-            Spacing="25"
-            Padding="30,0"
-            VerticalOptions="Center">
-
+        <VerticalStackLayout Padding="30,0" Spacing="25">
```

Comment:
It is not important, but I find it weird to have all the properties on the same line here when in the rest we use line breaks.

Reply:
I had the same thoughts in mind and have been working on it!